app/_locales/en/messages.json

```diff
@@ -191,6 +194,13 @@
     "message": "must be greater than or equal to $1 and less than or equal to $2.",
     "description": "helper for inputting hex as decimal input"
   },
+  "blockExplorerUrl": {
+    "message": "Block Explorer"
+  },
+  "blockExplorerView": {
+    "message": "View account at $1",
```

Comment:
at vs on?
```suggestion
    "message": "View account on $1",
```

Reply:
@danjm addressed a similar comment below and I believe `$1` will also be a URL here:

> I don't know what the correct grammar is here. $1 will always be a [URL]. "View at http://example.com" sounds a bit more correct... but I don't know why and I might be wrong.

I believe that Dan's correct here and that "at" sounds more natural.